test/cadmium_dynamic_LI_test.cpp

```diff
@@ -33,15 +33,15 @@
 
 #include "test_helpers.hpp"
 #include "../src/dynamic/LI_generator.cpp"
-#include "../../DEVSDiagrammer/model_json_exporter/include/dynamic_json_exporter.hpp"
+#include "../DEVSDiagrammer/model_json_exporter/include/dynamic_json_exporter.hpp"
```

Comment:
We should use <> inclusion here. It is an external library even if we are the ones developing it.